netlify.toml

```diff
@@ -188,4 +188,13 @@
 
 [[redirects]]
     from = "/docs/configuring-posthog"
-    to = "/docs/self-hosting/overview#configure"
\ No newline at end of file
+    to = "/docs/self-hosting/overview#configure"
+    
+[[redirects]]
```

Comment:
This is already handled [here](https://github.com/PostHog/posthog.com/pull/1603/commits/d00dfa501d8324befe5a783a4c3a74cbed542f87#diff-ab8f79b68b7adff7a07db953bf453f3c5aa6ade98d2b1b67d8432b36392489edR185) but the check didn't catch that.